packages/back-end/src/api/openapi/openapi.yaml

```diff
@@ -170,8 +173,14 @@ paths:
       $ref: "./paths/getExperiment.yaml"
     post:
       $ref: "./paths/updateExperiment.yaml"
+  /experiments/{id}/snapshot:
```

Comment:
Should this be plural?

Reply:
I'm not sure. It is for a single snapshot for the experiment, not all the experiments snapshots, so I feel like this is more grammatically correct. but idk if we have a strong preference.

Reply:
It should be plural no sure why I suggested otherwise.. This is the pattern for all other endpoints already.